### DISCUSSION TITLE
New to Github?

### DISCUSSION THREAD

**Discussion by kenjazz** [community/community#121533] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

If you are new to Github or have the passion to help newcomers, Please do so  undert this category

**Comment by ebndev** [comment]
Hi @kenjazz, thanks for being a part of the GitHub Community!

I encourage you to check out our [New to GitHub](https://github.com/orgs/community/discussions/categories/new-to-github) and [Programming Help](https://github.com/orgs/community/discussions/categories/programming-help) categories - you may find folks aligned with what you're looking for there 😄
